nixos/modules/programs/kubeswitch.nix

```diff
@@ -0,0 +1,54 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
```

Comment:
```suggestion
```

Reply:
I've seen it done in other packages as it makes fewer repetitions of `lib` in the whole file.

Reply:
or you can inherit the functions you're using specifically

Reply:
@itzwam Can you elaborate more please? Or a suggestion for a change?
Thanks in advance :D

Reply:
`nested with` can create issues in static analysis and can cause issues that are hard to diagnose. I'd recommend removing it.